### PR TITLE
Pdf\AbstractPdf::insertLogo() would override the  property from Pdf\I…

### PR DESCRIPTION
**Pull request by imadphp**
When overriding `\Magento\Sales\Model\Order\Pdf\Invoice::newPage()` to your custom module - changing `$this->y = 800` in the method to for example `$this->y=1000` in order to set the header to a higher y-position, the change is later overwritten by `\Magento\Sales\Model\Order\Pdf\AbstractPdf::insertLogo()`

### Description
`\Magento\Sales\Model\Order\Pdf\AbstractPdf::insertLogo()` should say `$top = $this->y` instead of overriding `$this->y` static number `830`.

### Steps to reproduce
1. Override `\Magento\Sales\Model\Order\Pdf\Invoice::newPage()` to your custom module.
2. Change `$this->y = 800` to for example `$this->y = 1000`.
3. Nothing happens, the header stays the same.

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=14000) <br/>All committers have signed the CLA.

**Comment by sidolov** [comment]
Hi @abbatis , describe issue what you are going to fix with steps to reproduce and please, sign CLA, otherwise, i can't process and accept your pull request.
Thank you

**Comment by magento-engcom-team** [comment]
@abbatis thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by hidotsh** [comment]
@sidolov CLA signed and PR updated

**Comment by sidolov** [comment]
Hi @abbatis , i verified proposed solution, described case works as expected, but default document for invoice entity has changed (top border is different). Also changed pdf files for other order entities. In that case, I propose to find another solution that can extend logic in `\Magento\Sales\Model\Order\Pdf\AbstractPdf::insertLogo()` regarding top border and leave all order document without any changes. 
Thanks!

**Comment by sidolov** [comment]
@abbatis , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration

**Comment by hidotsh** [comment]
@sidolov Hello, i've updated and reopened this pull request. I removed the 'needs update' label in case it prevents you from seeing this update. If this is wrong, please let me know. Thanks in advance.

**Review by orlangur — CHANGES_REQUESTED** [review]
Nowadays all changes must be applied to `2.3-develop` first (see https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#rules). Please prepare a new pull request, this one should be `on hold` until the latter is merged.

**Comment by orlangur** [comment]
Closing due to inactivity. Feel free to reach me out anytime later if you wish to continue work on this pull request and it will be reopened.
Thanks for collaboration!

**Comment by abbatis** [comment]
@orlangur I changed the destination for this pull request to `2.3-develop`.

**Comment by abbatis** [comment]
Never mind.

**Pull request closed without merging**
